app/controllers/content_types/school_home_pages_controller.rb

```diff
@@ -33,7 +33,7 @@ def slideshow
         'COMMON STYLES' => cascade_block('_cascade/blocks/html/common_styles'),
         'COMMON SCRIPTS' => cascade_block('_cascade/blocks/html/common_scripts'),
         'CUSTOM THEME' => cascade_block('_cascade/blocks/custom_themes/law'),
-        'GLOBAL FOOTER' => cascade_block('_cascade/blocks/html/law_footer'),
+        'GLOBAL FOOTER' => cascade_block('widgets/shared/law_footer'),
```

Comment:
This is probably okay for now, in the future i think they should both be under `_cascade/blocks/html/` if thats how it is in cascade also because I don't think that the footer is technically a widget?